saml_service_provider/settings.py

```diff
@@ -166,9 +166,9 @@ def __init__(
 
 
 class OneloginServiceProviderSettings(SAMLServiceProviderSettings):
-    def __init__(self, onelogin_connector_id=None, onelogin_x509_cert=None, onelogin_x509_fingerprint=None, **kwargs):
+    def __init__(self, onelogin_connector_id=None, onelogin_idp_entity_id=None, onelogin_x509_cert=None, onelogin_x509_fingerprint=None, **kwargs):
         kwargs['idp_metadata_url'] = 'https://app.onelogin.com/saml/metadata/{id}'.format(
-            id=onelogin_connector_id
+            id=onelogin_idp_entity_id
```

Comment:
I realize that this contradicts what I said in person, but I'm realizing now that it may be more robust to use `onelogin_connect_id` here if `onelogin_idp_entity_id` is `None`:
```suggestion
            id=onelogin_idp_entity_id or onelogin_connector_id
```

That way, a developer can use the constructor of `OneloginServiceProviderSettings` directly, instead of only by calling `get_onelogin_settings()`.

If we change this, that would make the logic below that sets the default of `onelogin_idp_entity_id` to `onelogin_connector_id` redundant, and we could simplify that.

Reply:
yep, this way is more elegant.